grails-app/i18n/messages.properties

```diff
@@ -3843,6 +3843,7 @@ react.combinedShipments.selectOrders.label=Select orders...
 react.combinedShipments.selectProduct.label=Select product...
 react.combinedShipments.availableItems.label=Available order items
 react.combinedShipments.blankTemplate.label=Blank import template
+react.combinedShipments.template.header.label=Order-items-template
```

Comment:
We have to remember that this one won't be testable for now

Reply:
This is for the name of the file that we generate, right? Why is the label `template.header.label` and not `template.fileName.label`? Or am I misunderstanding

Reply:
@EWaterman yeah, good point, all the time I had in mind the "header" word, as I was translating the headers in the template, but here, this definitely should be `fileName`.